openslides/agenda/models.py

```diff
@@ -344,6 +344,10 @@ def add(self, user, item, skip_autoupdate=False):
             raise OpenSlidesError(f"{user} is already on the list of speakers.")
         if isinstance(user, AnonymousUser):
             raise OpenSlidesError("An anonymous user can not be on lists of speakers.")
+        if config["agenda_present_speakers_only"] and not user.is_present:
+            raise OpenSlidesError("Only present users can be on the lists of speakers.")
+        if self.filter(user=user, item=item, begin_time=None).exists():
```

Comment:
This seems to be a rebase issue: Please remove these lines (they are 6 lines above)..